README.md

```diff
@@ -1,2 +1,159 @@
-# module-1-capstone-project
-My first capstone project at Microverse.
+<a name="readme-top"></a>
+# 📗 Table of Contents
+
+- [� Table of Contents](#-table-of-contents)
+- [📖 WOMEN IN TECH MENTORSHIP PROGRAM ](#-women-intech-)
+  - [🛠 Built With ](#-built-with-)
+    - [Tech Stack ](#tech-stack-)
+    - [Live Server](#live-server)
+    - [Key Features ](#key-features-)
+  - [💻 Getting Started ](#-getting-started-)
+    - [Prerequisites](#prerequisites)
+    - [Setup](#setup)
+    - [Install](#install)
+    - [Usage](#usage)
+    - [Run tests](#run-tests)
+    - [Deployment](#deployment)
+  - [👥 Authors ](#-authors-)
+  - [🔭 Future Features ](#-future-features-)
+  - [🤝 Contributing ](#-contributing-)
+  - [⭐ Show your support ](#-show-your-support-)
+  - [🙏 Acknowledgments ](#-acknowledgments-)
+  - [📝 License ](#-license-)
+
+<!-- PROJECT DESCRIPTION -->
+
+# 📖 WOMEN IN TECH MENTORSHIP PROGRAM. <a name="about-project"></a>
+
+*Women in Tech Mentorship Program* is a website that gives more information about the Women in Tech Annual Mentorship program. It is my first microverse capstone project.
+
+## 🛠 Built With <a name="built-with"></a>
+
+### Tech Stack <a name="tech-stack"></a>
+
+<details>
+  <summary>Client</summary>
+  <ul>
+    <li><a href="https://www.w3schools.com/html/">HTML</a></li>
+    <li><a href="https://www.w3schools.com/css/">CSS</a></li>
+  </ul>
+</details>
+
+<br />
+
+### Live Demo <a name="Live-server"></a>
+
+[WOMEN IN TECH WEBSITE](https://www.loom.com/share/85d6ee4122a74986895ca5dd81e36352)
```

Comment:
- [ ] Great job adding a live video, but note that you will make a presentation later on this. So for now, kindly make use of GitHub Pages to deploy the web pages.
- ensure The project is deployed properly, all pages, links, and images work as expected.
-  finally, please add A link to the deployed project in the README.

Reply:
Thank you so much for your review, I have deployed the web pages and fixed the link.